src/components/GobanThemePicker/GobanThemePicker.styl

```diff
@@ -28,12 +28,15 @@
         flex-direction: row;
         flex-wrap: wrap;
         justify-content: center;
+		text-align: center;
```

Comment:
I see you converted tabs to spaces in the .ts files but looks like the .styl file was missed, spaces here too please :)

Reply:
Curses and appologies, should be fixed.